syncer/filter.go

```diff
@@ -128,7 +110,7 @@ func (s *Syncer) skipQuery(tables []*filter.Table, stmt ast.StmtNode, sql string
 
 	if len(tables) > 0 {
 		tbs := s.bwList.ApplyOn(tables)
-		if len(tbs) == 0 {
+		if len(tbs) != len(tables) {
```

Comment:
suppose we rename a table from name `old` to name `new`, and table `old` is in `do-tables` and `new` is new `ignore-tables`.
should we filter this rename DDL?

Reply:
yep